src/postman/transformers/__tests__/request.test.ts

```diff
@@ -0,0 +1,351 @@
+      expect(transformHeader(new Header({ key: 'testKey', value: 'some string' }))).toEqual({
+        name: 'testKey',
+        schema: {
+          type: 'string',
+        },
+        style: 'simple',
+        examples: [
+          {
+            key: 'default',
+            value: 'some string',
+          },
+        ],
+      });
+    });
+  });
+
+  describe('value is not defined', () => {
+    it('results does not contain schema', () => {
+      expect(transformHeader(new Header({ key: 'testKey' }))).toEqual({
+        name: 'testKey',
+        style: 'simple',
+      });
+    });
+  });
+});
+
+describe('transformPathParams()', () => {
+  it('transforms correctly', () => {
+    expect(transformPathParams(['elem1', ':param1', ':param2', 'elem2'])).toEqual([
+      { name: 'param1', style: 'simple' },
+      { name: 'param2', style: 'simple' },
+    ]);
+  });
+});
+
+describe('transformBody()', () => {
+  describe('body is passed in raw mode', () => {
+    describe('body is defined', () => {
+      describe('mediaType is JSON-ish', () => {
+        describe('body is correctly defined json', () => {
+          expect(transformBody(new RequestBody({ mode: 'raw', raw: '{"a":"b"}' }), 'application/nice+json')).toEqual({
+            contents: [
+              {
+                examples: [{ key: 'default', value: { a: 'b' } }],
+                mediaType: 'application/nice+json',
+                schema: {
+                  properties: { a: { type: 'string' } },
+                  type: 'object',
+                },
+              },
+            ],
+          });
+        });
+
+        describe('body is not a correct JSON', () => {
+          expect(transformBody(new RequestBody({ mode: 'raw', raw: '"a":"b"' }), 'application/json')).toEqual({
+            contents: [
+              {
+                examples: [{ key: 'default', value: '"a":"b"' }],
+                mediaType: 'application/json',
+              },
+            ],
+          });
+        });
+      });
+
+      describe('mediaType is not JSON-ish', () => {
+        describe('media type is defined', () => {
+          it('returns body containing example and given media type', () => {
+            expect(transformBody(new RequestBody({ mode: 'raw', raw: '<a />' }), 'application/xml')).toEqual({
+              contents: [
+                {
+                  examples: [{ key: 'default', value: '<a />' }],
+                  mediaType: 'application/xml',
+                },
+              ],
+            });
+          });
+        });
+
+        describe('media type is not defined', () => {
+          it('returns body containing example and text/plain media type', () => {
+            expect(transformBody(new RequestBody({ mode: 'raw', raw: "I'm a goat. Bleeet!" }))).toEqual({
+              contents: [
+                {
+                  examples: [{ key: 'default', value: "I'm a goat. Bleeet!" }],
+                  mediaType: 'text/plain',
+                },
```

Comment:
This case should not technically be possible but I am ok with the fallback.

Reply:
Well, it is:
<img width="938" alt="Screen Shot 2020-02-05 at 14 26 33" src="https://user-images.githubusercontent.com/7136002/73845731-8b849a80-4823-11ea-934b-d074416ee4f8.png">

In the Postman app, first I created a normal request and the explicitly removed `Content-type` header.